calico/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+calico-ctl:
+  image: calico/ctl
+  net: host
+  command: node --runtime=none
```

Comment:
I believe there is, but that it doesn't matter.  The restart mode will handle any race conditions, if they are actually hit (which I haven't seen).